cmake_modules/TokuSetupCompiler.cmake

```diff
@@ -192,14 +199,18 @@ endif ()
 set(CMAKE_C_FLAGS "-Wall -Werror ${CMAKE_C_FLAGS}")
 set(CMAKE_CXX_FLAGS "-Wall -Werror ${CMAKE_CXX_FLAGS}")
 
+## It looks like there is no more need in these changes for at least
+## MacOS 10.11, so the followed code is just commented out, but it's
+## still here just in case.
+##
 ## need to set -stdlib=libc++ to get real c++11 support on darwin
-if (APPLE)
-  if (CMAKE_GENERATOR STREQUAL Xcode)
-    set(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LIBRARY "libc++")
-  else ()
-    add_definitions(-stdlib=libc++)
-  endif ()
-endif ()
+#if (APPLE)
+#  if (CMAKE_GENERATOR STREQUAL Xcode)
+#    set(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LIBRARY "libc++")
+#  else ()
+#    add_definitions(-stdlib=libc++)
+#  endif ()
+#endif ()
```

Comment:
Similar to Laurynas suggestion, I would just remove all of this commented code here.